src/redfishpower/redfishpower.c

```diff
@@ -75,7 +75,8 @@ static zhashx_t *test_power_status;
 /* in usec
  *
  * status polling interval of 1 second may seem long, but testing
- * shows wait ranges from a few seconds to 20 seconds
+ * shows wait for on/off to complete ranges from a few seconds to 50
+ * seconds
```

Comment:
Since this test is very hardware specific the specific hardware/firmware should be mentioned.